### DISCUSSION TITLE
Dealing with CORS

### DISCUSSION THREAD

**Discussion by prologic** [authelia/authelia#5549] [Troubleshooting] [answered]
Hi,

This is an example request coming from Rainloop Webmail that I see in the Chrome Console:

```
Access to manifest at 'https://auth.domain.tld/?rd=https%3A%2F%2Fwebmail.domain.tld%2Frainloop%2Fv%2F1.13.0%2Fstatic%2Fmanifest.json%3Fcommunity&rm=GET' (redirected from 'https://webmail.domain.tld/rainloop/v/1.13.0/static/manifest.json?community') from origin 'https://webmail.domain.tld' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

How do I go about configuring Authelia rules appropriately for this?

**Comment by james-d-elliott** [comment, marked as answer]
Sorry for the delay in my response. 

I believe this is generally caused when an application tries to load the page provided in a 30x response via javascript instead of redirecting the user. I believe we've removed the CORS policy before in these instances which doesn't result in any clear victory. However you're welcome to strip the CORS headers out and report back too. If it resolves it then we can maybe include such an option.
